config/logging.rb

```diff
@@ -44,7 +44,7 @@
   config.lograge.custom_options = ->(event) do
     # show params for every request
     unwanted_keys = ['format', 'action', 'controller']
-    params = event.payload[:params].reject { |key, _| unwanted_keys.include? key }
+    params = event.payload[:params]&.reject { |key, _| unwanted_keys.include? key }
```

Comment:
looks a bit sus ... make sure it still logs other params fine

Reply:
I originally added this in because we're getting a bunch of errors that event.payload[params] is Nil. I'll dig in and see if this is going to cause more problems than it solves.

Reply:
Tested this in staging and it didn't fix it anyway 😆 . Reverting.